src/Microsoft.Win32.Registry/pkg/Microsoft.Win32.Registry.pkgproj

```diff
@@ -3,12 +3,18 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\Microsoft.Win32.Registry.csproj">
-      <SupportedFramework>net461;netcoreapp2.0</SupportedFramework>
+      <SupportedFramework>net461;netcoreapp2.0;uap10.1;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
Yeah, I dealt with that in the rebase.  UAP will still be supported, only now with a PlatformNotSupported assembly.